Remotion/ObjectBinding/Web/UI/Controls/BocListImplementation/Rendering/BocRowRenderer.cs

```diff
@@ -188,6 +205,48 @@ private IBocIndexColumnRenderer GetIndexColumnRenderer ()
       return _indexColumnRenderer;
     }
 
+    private void RenderValidationRow (
+        BocListRenderingContext renderingContext,
+        BocListRowRenderingContext rowRenderingContext,
+        BocListDataRowRenderEventArgs rowRenderEventArgs,
+        IReadOnlyCollection<BocListValidationFailureWithLocationInformation> validationFailuresWithLocationInfo,
+        int rowIndex)
+    {
+      var cssClassValidationRow = GetCssClassValidationRow(rowRenderingContext.IsSelected, rowRenderEventArgs);
+
+      renderingContext.Writer.AddAttribute(HtmlTextWriterAttribute.Class, cssClassValidationRow);
+      renderingContext.Writer.RenderBeginTag(HtmlTextWriterTag.Tr);
+
+      var columnRenderers = renderingContext.ColumnRenderers;
+
+      var validationColumnRenderer = columnRenderers.First(r => r.ColumnDefinition is BocValidationErrorIndicatorColumnDefinition);
+
+      var amountColumnsLeftOfValidation = validationColumnRenderer.VisibleColumnIndex;
```

Comment:
`numberOfColumnsBeforeValidationColumn`

'amount' is usually a sum result, not a count.
'LeftOfValidation' could also be read as some sort of leftover value. There's also the semantic detail that left/right are only helpful in LTR writing. If the BocList is displayed in RTL writing, the semantics of left/right are inverted so before/after are more precise.